value/src/main/java/com/google/auto/value/extension/AutoValueExtension.java

```diff
@@ -18,7 +18,8 @@
  *
  * <p>Extensions can extend the AutoValue implementation by generating subclasses of the AutoValue
  * generated class. It is not guaranteed that an Extension's generated class will be the final
- * class in the inheritance hierarchy, unless its {@link #mustBeFinal(Context)} method returns true.
+ * class in the inheritance hierarchy, unless its
+ * {@link com.google.auto.value.extension.AutoValueExtension#mustBeFinal(Context)} method returns true.
```

Comment:
Yeah, I'm not sure why those got in there.  Works fine for me, too.  I've reverted them.